.github/workflows/deploy-nuxeo.com-benchmarks.yml

```diff
@@ -4,14 +4,36 @@ on:
   push:
     branches:
       - master
+  workflow_dispatch:  # Manual triggering
 jobs:
   build-and-deploy:
-    runs-on: benchmarks
+    runs-on: ubuntu-latest
+    environment:
+      name: ${{ github.ref_name == 'master' && 'production'  || (startsWith(github.ref_name, 'NXDEVOPS') && 'NXDEVOPS') || github.ref_name }}
```

Comment:
If I create a branch called `production` it will deploy to production which we don't want.
Also it would be more natural to have an env called development instead of NXDEVOPS imho.